### PR TITLE
Laboratory work #4, Andrew Zapekin - 23FPL1

### PR DESCRIPTION
**Pull request by Andrew-cmd216**

### CODE REVIEW

**Comment by EIzharikov** [comment]
Nothing to review, I hope at least the memes will be funny

**Review by EIzharikov — COMMENTED** [review]
You should hurry up! Make CI green and continue your work!

**Comment by EIzharikov** [comment]
nothing new to review

**Comment by EIzharikov** [comment]
Nothing new to review, CI is red (!). Good luck for the next Friday

File: lab_4_retrieval_w_clustering/main.py [hunk 1]
```diff
@@ -51,6 +59,16 @@ def set_tokenized_corpus(self, tokenized_corpus: TokenizedCorpus) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not tokenized_corpus or not isinstance(tokenized_corpus, list):
+            raise ValueError()
+
+        self._corpus = tokenized_corpus
+        sum = 0
+
+        for index, paragraph in enumerate(self._corpus):
```
[the remark this answers is not in the document]
Comment (Andrew-cmd216) [reply lab_4_retrieval_w_clustering/main.py]: done

File: lab_4_retrieval_w_clustering/main.py [hunk 2]
```diff
@@ -26,6 +26,11 @@ def get_paragraphs(text: str) -> list[str]:
     Returns:
         list[str]: Paragraphs from document.
     """
+    if not text or not isinstance(text, str):
+        raise ValueError()
+    paragraphs = text.split('\n')
```
[the remark this answers is not in the document]
Comment (Andrew-cmd216) [reply lab_4_retrieval_w_clustering/main.py]: done

**Pull request closed without merging**